bundles/edu.kit.ipd.sdq.pcm.simulation.scheduler.exact/src/edu/kit/ipd/sdq/pcm/simulation/scheduler/exact/SimActiveResource.java

```diff
@@ -40,8 +41,9 @@
     public static final Logger logger = Logger.getLogger("Scheduler");
 
 
-    public SimActiveResource(final ExactSchedulingFactory exactSchedulingFactory, final SchedulerModel model, final ActiveResourceConfiguration resourceConf) {
-        super(model, resourceConf.getReplicas(), resourceConf.getName(), resourceConf.getId());
+    public SimActiveResource(final ExactSchedulingFactory exactSchedulingFactory, final SchedulerModel model, final ActiveResourceConfiguration resourceConf
+            , IResourceTableManager resourceTableManager) {

```

Comment:
Same

Reply:
worked in review finding; reformatted file using Palladio code formatter